### PR TITLE
📝 Doc conventions, 🔨 issue templates and ♻️ error messages

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- 📝 doc conventions
- 🔨 template issues and tools for releases (originally in ITISFoundation/osparc-issues). This is a prerequisite before all the release issues are moved to this repo
    -  Converts markdown to forms https://github.com/pcrespov/sandbox-python/issues/new/choose
- ♻️ Following up pair-debugging with @odeimaiz : makes invitation error more human-readable

## Related issue/s

-  #3943: adds pre-release issue template to this repo

## How to test

 - check to create in issue using  https://github.com/pcrespov/sandbox-python/issues/new/choose

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2_feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: ✨ Feature request
+description: Suggest an idea to implement in the simcore plaform
+labels: ["t:enhancement"]
+assignees: ["pcrespov"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
```
Comment (sanderegg) [inline .github/ISSUE_TEMPLATE/2_feature_request.yml:9]: it's a feature request, not a bug

File: .github/ISSUE_TEMPLATE/3_maintenance_issue.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: 🏗️ Maintenance
+description: A change in the code to overcome technical debt
+labels: ["t:maintenance"]
+assignees: ["pcrespov"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
```
Comment (sanderegg) [inline .github/ISSUE_TEMPLATE/3_maintenance_issue.yml:9]: also not a bug I guess?

File: .github/ISSUE_TEMPLATE/3_maintenance_issue.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Planned changes
+      description: |
+        After describing the changes, please connect tohttps://github.com/ITISFoundation/osparc-issues/issues/675
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+      value: Please connect this case to the Epic https://github.com/ITISFoundation/osparc-issues/issues/675
```
Comment (sanderegg) [inline .github/ISSUE_TEMPLATE/3_maintenance_issue.yml:20]: maybe it can also be automatically related to it? so at least there will be an entry.

**Review by sanderegg — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/2_feature_request.yml (hunk 1 above)
Comment (pcrespov) [reply .github/ISSUE_TEMPLATE/2_feature_request.yml:9]: :-D the dangers of copy&aste

File: .github/ISSUE_TEMPLATE/3_maintenance_issue.yml (hunk 3 above)
Comment (pcrespov) [reply .github/ISSUE_TEMPLATE/3_maintenance_issue.yml:20]: NO, unfortunately no. I will try that in a second round where these PRs could be created via the API

**Review by mrnicegyu11 — APPROVED** [review]
Thanks a lot for this!

**Pull request merged**